getting-started/proc-add-custom-app-file-openshift.adoc

```diff
@@ -30,15 +30,28 @@ data:
 . Go to the *Helm* tab.
 +
 The list of Helm Releases appears on the page.
-. Click the three dots of a Helm release and select *Upgrade*.
-. Add the following values in *Root Schema -> Backstage Chart Schema ->  Backstage Parameters -> Extra App Configuration files to inline into command arguments*:
+. Click the three dots corresponding to a Helm release and select *Upgrade*.
+. Use one of the following views to edit the Helm configuration:
+
+** Using *Form view* 
++
+.. Expand *Root Schema → Backstage chart schema → Backstage parameters → Extra app configuration files to inline into command arguments*. 
+.. Click the *Add Extra app configuration files to inline into command arguments* link.
+.. Enter the value in the following fields:
 +
 --
-* *ConfigMapRef*: `app-config-rhdh`
+* *configMapRef*: `app-config-rhdh`
 * *filename*: `app-config-rhdh.yaml`
 --
-. Click *Upgrade*.
-
+.. Click *Upgrade*.
 
+** Using *YAML view* 
 
+.. Set the value of the following parameters:
```

Comment:
This visualization is not easy to understand as the YAML view shows the full manifest of the Helm Chart, so it should be better showing the yaml structure of this change. For example:

```yaml
# ... other Developer Hub Helm Chart configuration
upstream:
  backstage:
    extraAppConfig:
      - configMapRef: app-config-rhdh
        filename: app-config-rhdh.yaml
# ... other Developer Hub Helm Chart configuration
```